src/main/java/nl/smallproject/www/techiteasy/repositories/TelevisionsRepository.java

```diff
@@ -0,0 +1,12 @@
+package nl.smallproject.www.techiteasy.repositories;
+
+import nl.smallproject.www.techiteasy.models.Television;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
```

Comment:
Deze annotatie is hier niet nodig. Je extend van de JpaRepository, daar wordt dit al geregeld.